pages/en/developers/apis/4-realtime-api/vehicle-positions/index.md

```diff
@@ -161,7 +162,11 @@ Go hog wild.
 
 ### Command line
 
-Below are sample subscriptions utilizing the MQTT.js command-line tools.
+Below are sample subscriptions utilizing [MQTT.js](https://github.com/mqttjs/MQTT.js) command line tools.  
+MQTT.js command line tools can be installed with:  
+```
+npm install -g mqtt
+```
 
```

Comment:
(The GitHub UI seems a bit weird for this suggestion. I'm not sure if it look okay before committing.)
```suggestion

If you insist on using mosquitto, try this for TLS access: `mosquitto_sub --capath "/etc/ssl/certs/" -h mqtt.hsl.fi -p 443 -v -t "/hfp/v1/journey/#"`
```
```

Reply:
It's just that some prefer Python over JS. In my understandin, mqtt.js is not in Linux distribution package repositories, mosquitto often is.